api/src/resolvers/banking/banking-resolver.ts

```diff
@@ -43,7 +44,9 @@ const checkIfLastServiceBanked = async (
       record.transactionStatus === 'success') &&
     record.id !== serviceRecordId
   ) {
-    throwErrorMsg('You cannot bank without banking your previous offering')
+    throwErrorMsg(
+      'Please bank last weeks outstanding offering before attempting to bank this weeks'
```

Comment:
Oh. The english and punctuation are not working in this sentence

Reply:
Changed